astropy/nddata/ccddata.py

```diff
@@ -522,7 +522,7 @@ def _generate_wcs_and_update_header(hdr):
         # cases (malformed header) it could fail...
         log.info(
             "An exception happened while extracting WCS information from "
-            "the Header.\n{}: {}".format(type(exc).__name__, str(exc))
+            f"the Header.\n{type(exc).__name__}: {str(exc)}"
```

Comment:
I thought `{a}` by default does `str(a)`, so `str` here would be superfluous.

Reply:
I thought so too. Ruff just doesn't have that built-in simplification yet.